backport-changelog/6.6/7012.md

```diff
@@ -0,0 +1,3 @@
+https://github.com/WordPress/wordpress-develop/pull/7012
+
+* https://github.com/WordPress/gutenberg/pull/63403
```

Comment:
As this PR is intended for a 6.6 point release if there isn't an RC4, I've point the backport changelog under 6.6. If that needs changing let me know.